javascript/CodeChallenges/CC-10_stack-and-queue/stack.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const Node = require('./node')
+
+class Stack {
+  constructor() {
+    // this.storage
+    this.top = null;
+  }
+  peek() {
+    if(!this.top){throw new Error('no peek for you');}
+    return this.top;
+  }
+  push(val) {
+    this.storage.unshift(val);
+    this.top = val;
+  }
+  pop() {
+    const thing = this.storage.shift():
+    this.top = this storage[0];
+    return thing;
+  }
+}
+
```

Comment:
Looks like you are off to a great start. I would like to see some tests added and a README as well!